app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -55,150 +55,103 @@ keywords:
 
-To apply, you must have been sponsored by a Home Office licensed student sponsor over the course of your studies.
+### Your partner and children
 
-The graduate visa will allow you to work or look for work in the UK for up to 2 years after completing your studies (3 years for PhD students) without a sponsor. This includes working as a teacher. 
+You may be able to [bring your partner and children with you on a Skilled Worker visa](https://www.gov.uk/skilled-worker-visa/your-partner-and-children).
 
-You will be able to switch from a graduate visa to another visa (such as the skilled worker visa) without having to leave the UK, if you meet the requirements for the new visa route.
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/graduate-visa">Apply for a graduate visa<span></span></a>
-</p>
+## Check your status
 
-## High potential individual visa
+If you’re not sure about your immigration status, you can [check if you need a UK visa](https://www.gov.uk/check-uk-visa).
 
-To be eligible for a high potential individual visa, you must:
+You will not need a visa to teach in the UK if you:
 
-* have a bachelor’s degree, master’s degree or PhD from a university listed in the [Global Universities](https://www.gov.uk/high-potential-individual-visa/eligibility) list
-* have been awarded your qualification no more than 5 years before you apply
-* have English language skills to [B1 intermediate level](https://www.efset.org/cefr/b1/)
-* have personal funds of £1,270
-* pass a security and criminality check
-
-You do not need an offer of a teaching job to apply for this visa and you’ll be able to stay in the UK for 2 years with a bachelor’s or master’s degree, and 3 years with a PhD. 
-
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/high-potential-individual-visa">Apply for a high potential individual visa<span></span></a>
-</p>
+* are an Irish citizen
 
-## India young professionals scheme
+* have settled or pre-settled status under the [EU Settlement Scheme (EUSS)](https://www.gov.uk/settled-status-eu-citizens-families)
 
-To be eligible for the India young professionals scheme visa, you must:
+* have [indefinite leave to remain](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk) 
 
-* be an Indian citizen
-* be between 18 and 30 years old
-* have an eligible qualification
-* have £2,530 in savings
+You will not need to be sponsored for a Skilled Worker visa if you have a: 
 
-The visa allows you to live and work in the UK for up to 2 years. To apply for the visa, you must first enter a ballot. 
+* [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa) 
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/india-young-professionals-scheme-visa">Apply for the India young professionals scheme<span></span></a>
-</p>
+* [Family visa](https://www.gov.uk/uk-family-visa) 
```

Comment:
I don't think this needs an upper case F but maybe keep in for consistency. Looks like the others do start with upper case!